R/drop_name_crossref.R

```diff
@@ -0,0 +1,46 @@
+#' Create visual citation from Crossref data
+#'
+#' Takes one or several dois and extracts information from Crossref, then processes them into visual citations with \code{\link{drop_name}}
+#'
+#' @param dois One or several dois to create visual citations for. If they are named, these names are used as filenames; otherwise they are generated based on authors and years.
+#' @inheritDotParams drop_name -bib -cite_key
+#'
```

Comment:
drop_name_crossref should behave like drop_name and return a file path to keep things consistent. I would add this to the documentation, like e.g.

`#' @return A character string with the file path to the created visual citation in the specified output format.`